### PR TITLE
initial github_traffic script commit.

### PR DESCRIPTION
**Pull request by milljm**
Initial commit. This will require an update to our moose-environment package to include the python-mechanize module for seamless use. Refs #5049

### CODE REVIEW

**Comment by dschwen** [comment]
Do we really need our users to run this?

**Comment by jwpeterson** [comment]
No, but we wanted to make the script publicly-available.

File: scripts/github_traffic.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+import cookielib
```
Comment (permcody) [inline scripts/github_traffic.py]: Mmm - cookie

File: scripts/github_traffic.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+except ImportError:
+  print 'python-mechanize module not available.\n'
+  sys.exit(1)
+
+def webBrowser():
+  # Browser
+  br = mechanize.Browser()
+
+  # Cookie Jar
+  cj = cookielib.LWPCookieJar()
```
Comment (permcody) [inline scripts/github_traffic.py]: Mmm - CookieJar!

File: scripts/github_traffic.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      print '\tError 406: You do not have permission to view statistics. Or you supplied incorrect credentials'
+      sys.exit(1)
+    if str(e).find('404') != -1:
+      print '\tError 404: Page not found'
+      sys.exit(1)
+  return stats
+
+def verifyArgs(args):
+  if args.repo is None or len(args.repo.split('/')) != 2:
+    print '\nYou must specify a repository you are insterested in scrapeing:\n\t --repo foo/bar\n\nNote: GitHub is case-sensitive, so your arguments must be too'
```
Comment (permcody) [inline scripts/github_traffic.py]: Type - "scraping"

**Comment by dschwen** [comment]
How could this possibly fail?

**Comment by jwpeterson** [comment]
> How could this possibly fail?

Accidental libmesh de-update

**Pull request merged**